sdk/core/azure-core/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### New Features
 
 - Added support for HTTP conditional requests `MatchConditions` and `RequestConditions`.
+- Added MD5 hashing APIs to the `Azure::Core` namespace available from `azure/core/md5.hpp`.
```

Comment:
I would rather like listing the added APIs than mentioning the md5.hpp file.
```suggestion
- Added MD5 hashing APIs to `Azure::Core` namespace: 
  -  Foo()
  -  Bar()
```

Reply:
This is still in flux, so deferring to later. Also, part of the reason for stating it this way was brevity to avoid bloating the CL.